### PR TITLE
Minor fixes for IPA dev cluster provision script and templates

### PR DESCRIPTION
**Pull request by denalex**
- updated terraform template and added designation of some values as sensitive to make it work with the latest versions of terraform
- added a flag to not error out when cleanup tries to remove files that might not exist during the first run

### CODE REVIEW

**Review by hpandeycodeit — APPROVED** [review]

File: concourse/terraform/ipa-multinode-hadoop/outputs.tf [hunk 1]
```diff
@@ -5,14 +5,16 @@ output "ssh_config" {
     namenode = google_compute_instance.namenode
     datanode = google_compute_instance.datanode
   })
-
+  sensitive = true
```
Comment (ashuka24) [inline concourse/terraform/ipa-multinode-hadoop/outputs.tf:8]: why is this needed?

**Review by ashuka24 — APPROVED** [review]

File: concourse/terraform/ipa-multinode-hadoop/outputs.tf (hunk 1 above)
Comment (denalex) [reply concourse/terraform/ipa-multinode-hadoop/outputs.tf:8]: without this the newer versions of terraform fail, they want us to explicitly declare the intention of saving a sensitive information to local disk to prevent accidental developer errors.

**Comment by edespino** [comment]
Closing this PR as it was incorporated into PR #899.
